Ryujinx.HLE/HOS/Services/Sockets/Bsd/IClient.cs

```diff
@@ -530,7 +530,16 @@ public ResultCode RecvFrom(ServiceCtx context)
 
                     receiveRegion.Dispose();
 
-                    context.Memory.Write(sockAddrOutPosition, BsdSockAddr.FromIPEndPoint(endPoint));
+                    var bsdSockAddr = BsdSockAddr.FromIPEndPoint(endPoint);
+
+                    if (sockAddrOutSize != 0 && sockAddrOutSize >= (ulong) Unsafe.SizeOf<BsdSockAddr>())
+                    {
+                        context.Memory.Write(sockAddrOutPosition, bsdSockAddr);
```

Comment:
```suggestion
                        context.Memory.Write(sockAddrOutPosition, BsdSockAddr.FromIPEndPoint(endPoint));
```

`bsdSockAddr` is a bit useless here since you don't use it elsewhere.

Reply:
Agree, its a left over from before when i had bsdSockAddr.Size()